tests/setup.ts

```diff
@@ -0,0 +1,34 @@
+import * as dotenv from 'dotenv';
+import { chromium } from '@playwright/test';
+import type { FullConfig, Browser } from '@playwright/test';
+
+dotenv.config();
+
+async function globalSetup(config: FullConfig) {
+    const { storageState } = config.projects[0].use;
+
+    const chromiumBrowser = await chromium.launch();
+    await setupPage(chromiumBrowser, storageState as string)
+}
+
+async function setupPage(browser: Browser, storageState: string) {
+    const page = await browser.newPage();
+    await page.goto(`${process.env.APPLICATION_BASE_URL}/logto/sign-in`);
```

Comment:
Isn't it better to go to Swagger page first, and click the Login button? That would be more accurate to what users do.